Add tip how to reset the db to setup guide - Closes #1099 (#1109)
LiskHQ/lisk-docs: modules/ROOT/pages/guides/app-development/setup.adoc

  start --network testnet --config ~/my_custom_config.json
----

.How to reset the database of an application
[TIP]

====
Once the application is started for the first time, it will save the application-specific data under the path `~/.lisk/new_app`.

To reset the database of the application, simply delete the folder with the application data:

[source,bash]
----
rm -r ~/.lisk/new_app/
----

> To remove the data of all application, execute:

Reviewer: ```suggestion
To remove all data from the application, execute the following command:
```

Author: ```suggestion
To remove the data of all applications, execute:
```

[source,bash]
----
rm -r ~/.lisk/
----
====

=== Next steps

> By installing Lisk Commander and running `lisk init`, you have now a working blockchain application with the default configurations for runnning in a local devnet.

Reviewer: ```suggestion
By installing Lisk Commander and running `lisk init`, a working blockchain application now exists with the default configurations for running in a local devnet.
```

  start --network testnet --config ~/my_custom_config.json
----

.How to reset the database of an application
[TIP]

====
Once the application is started for the first time, it will save the application-specific data under the path `~/.lisk/new_app`.

To reset the database of the application, simply delete the folder with the application data:

[source,bash]
----
rm -r ~/.lisk/new_app/
----

> To remove the data of all applications, execute:

Reviewer: ```suggestion
To remove the data from all applications, execute:
```
